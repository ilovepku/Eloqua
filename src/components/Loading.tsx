import React from 'react'
import {View, ActivityIndicator} from 'react-native'
import tailwind from 'tailwind-rn'

const Loading: React.FC = () => (
  <View style={tailwind('flex-1 bg-white items-center justify-center')}>
    <ActivityIndicator size="large" color="#42a5f5" />
  </View>
)

export default Loading
