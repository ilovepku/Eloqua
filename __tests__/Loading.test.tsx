import React from 'react';
import {render} from '@testing-library/react-native';

import Loading from '../src/components/loading/Loading';

describe('Loading', () => {
  it('renders correctly', () => {
    const tree = render(<Loading />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
