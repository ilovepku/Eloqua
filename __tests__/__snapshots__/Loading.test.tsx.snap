// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgba(255, 255, 255, 1)",
      "flexBasis": "0%",
      "flexGrow": 1,
      "flexShrink": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#42a5f5"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
